sdk/inc/azure/core/az_mqtt5_rpc_client.h

```diff
@@ -43,6 +43,22 @@ typedef struct
    */
   uint32_t publish_timeout_in_seconds;
 
+  /**
+   * @brief The topic format to use for the subscription topic.
+   *
+   * @note Can include {name} for command name, {serviceId} for model id, {executorId} for the
+   * server's client_id, and/or {invokerId} for the client's client_id
```

Comment:
Some of these end with a period, and others do not. What is the consistency here?